rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -774,6 +780,9 @@ namespace np
 			string_to_npid(s_npid, npid);
 			auto& sigh = g_fxo->get<named_thread<signaling_handler>>();
 			sigh.set_self_sig_info(npid);
+
+			const std::string s_country = g_cfg_rpcn.get_country();
+			string_to_country_code(s_country, country_code);
```

Comment:
Once the setting is moved away from the RPCN config, it should probably be done unconditionally and not only when RPCN is active.